wolfcrypt/src/sp_int.c

```diff
@@ -17888,12 +17878,43 @@ int sp_read_unsigned_bin(sp_int* a, const byte* in, word32 inSz)
     #endif /* LITTLE_ENDIAN_ORDER */
         }
 #endif
-        sp_clamp_ct(a);
+        if (! ct)
+            sp_clamp_ct(a);
```

Comment:
sp_clamp_ct was being called.
Always call it unless there is a real need.
We have to have a clamp.

Don't see a need for the new API if this is changed back.

Reply:
Ignore this, I misunderstood.

Reply:
fixed (new API removed).